crates/matrix-sdk/src/client/mod.rs

```diff
@@ -266,6 +266,22 @@ impl Client {
         Ok(())
     }
 
+    /// Process a [transaction] received from the homeserver which has already
+    /// been converted into a sync response.
+    ///
+    /// [transaction]: https://matrix.org/docs/spec/application_service/r0.1.2#put-matrix-app-v1-transactions-txnid
+    #[cfg(feature = "appservice")]
+    pub async fn receive_preprocessed_transaction(
```

Comment:
Does this obsolete the `receive_transaction()` method? If not could you add some info how this method differs from the `receive_transaction()` one, cross linking from one to the other would be great as well.

Reply:
I hadn't considered that, there probably isn't much point in keeping the old version of the method

Reply:
Let's remove it then.